diagnostics/README.md

```diff
@@ -0,0 +1,70 @@
+# Diagnostics tools
+
+A collection of tools to help triage and diagnose misbehaving networks.
+
+Notation used:
+* TensorFlow: TF
+* TensorBoard: TB
+* protobuf binary: pb
+* protobuf string text: pbtxt
+* NGraph-TF: NGTF
+
+## Debug flags
+* ```NGRAPH_ENABLE_SERIALIZE=1```: Generate nGraph level serialized graphs .json
+* ```NGRAPH_CPU_TRACING=1```: Generate nGraph level function timelines
+* ```NGRAPH_VLOG_LEVEL=5```: Generate ngraph-tf logging info for different passes
+* ```NGRAPH_GENERATE_GRAPHS_PBTXT=1```: Generate .pbtxt files for different phases in ngraph-tf bridge
+* ```NGRAPH_TF_LOG_PLACEMENT=1```: will generate op placement log to stdout
+* ```NGRAPH_TF_VALIDATE_CLUSTER_GRAPHS=1```: Dumps Encapsulated TF Graphs: ngraph_cluster_<cluster_num>
```

Comment:
We need to rename this flag (in the C++ code) to something like: NGRAPH_TF_DUMP_CLUSTERS or something and update the doc here.

Reply:
Added a TODO for self